src/components/Missions.js

```diff
@@ -25,23 +26,23 @@ const Missions = () => {
             <th>{}</th>
           </tr>
         </thead>
-        <tbody d-flex>
+        <tbody>
           {selectedMissions.map((m) => (
             <tr key={uuidv4()}>
+              {console.log(m.isReserved)}
               <td className="mission-name">{m.mission_name}</td>
```

Comment:
- [ ] (OPTIONAL) It is good practice to remove all `console.log` from your code.